.changelog/2e03eae07dff4beb94963939b6549426.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "2e03eae0-7dff-4beb-9496-3939b6549426",
+    "type": "bugfix",
+    "description": "Fix nil dereference panic for operations that require checksums, but do not have an input setting for which algorithm to use.",
+    "modules": [
+        "service/internal/checksum"
```

Comment:
should we add service/s3 here?

Reply:
It doesn't actually affect service/s3 code, so I'm not sure. I'd rather not in the context of this emergency patch in case it causes some sort of release tooling issue.

Reply:
nope, the fix don't include s3 itself

Reply:
OK I'm down to not have it. 

For context, what I was thinking was "where would someone think of looking to see we pushed a fix?"